buildSrc/src/main/java/com/uber/okbuck/core/dependency/exporter/ExporterException.java

```diff
@@ -0,0 +1,10 @@
+package com.uber.okbuck.core.dependency.exporter;
+
+import java.io.IOException;
+
+public class ExporterException extends RuntimeException {
```

Comment:
could extend from `IOException` and can remove the default constructor

Reply:
I switched to accept an Exception in the constructore. IOException is a checked exception, extending that will force the caller to handle the exception.